doc/installation.rst

```diff
@@ -34,25 +38,35 @@ Just visit the `Spyder page on Binder`_ to get started using Spyder now.
 
 
 
-.. _standalone_installers_ref:
+.. _install-standalone:
 
 =====================
 Standalone installers
 =====================
 
-Our standalone installers for Windows and macOS are available from Spyder 4.2 onwards.
-We recommend using this installation method on those platforms, but we offer several other options for Linux, advanced users and specific needs, so keep reading if that's the case for you.
+The standalone installers are our recommended method for most users on Windows and macOS, with experimental Linux support under active development.
+They work much like installing any other IDE, where Spyder can be installed and updated independently of the Python environments you use to run your code, avoiding problems with incompatible packages and broken installations that users often face when doing so within their own Python installation.
+
+The installers include a built-in Python environment with the most common scientific libraries, which can be used out of the box for basic data analysis tasks.
+However, to install and manage your own packages and environments, you'll currently need to use an external Python installation (such as `Anaconda`_, `Miniconda`_, `Miniforge/Mambaforge`_, `WinPython`_ or `Python.org <Python_>`__) with Spyder.
```

Comment:
```suggestion
However, to install and manage your own packages and environments, you'll currently need to use an external Python installation (such as `Anaconda`_, `Miniconda`_, `Miniforge/Mambaforge`_, `WinPython`_ or `Python.org <Python_>`__), which needs to be connected to our installers.
```

I think `with Spyder` would lead people to install Spyder on that installation, which we know is not necessary.

Reply:
```suggestion
However, to manage your own packages and environments, you'll currently need to connect an external Python distribution (such as `Anaconda`_, `Miniconda`_, `Miniforge/Mambaforge`_, `WinPython`_ or `Python.org <Python_>`__) to your standalone copy of Spyder.
```

* Avoid triple-repeat of installation
* Be more concise/less redundant